### DISCUSSION TITLE
10 luglio 2023 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#43] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/50ae51d5-b719-4d49-87bd-80ba67d4a13a)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/c0854799-9998-4c51-96b9-f91306e22d04)

> [!TIP]
> Se non sai come calcolare il logaritmo in base 2 di numeri molto grandi, sulla calcolatrice basta calcolare $\frac{\ln(\text{numero molto grande})}{\ln(2)}$

**Reply by niccolo-fato** [reply]
Alla fine come fa a venirti 8724672 non dovrebbe venire 6684672?

**Reply by IonutCicio** [reply]
Hai ragione, ho sbagliato a fare il conto 😅

**Reply by interestingmaneuver** [reply]
Calcolatrice? Pfft... si fa $\times 2$ finché non viene troppo grande...

**Reply by niccolo-fato** [reply]
🗿

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 17:00:46](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/17ebb3da-5d4d-4ec2-a5ab-08e5c9ece317)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/da461ad7-41c9-4a06-adf7-586a0b79f7b5)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/315943c5-2127-4476-80f1-d3254dafd495)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/62ad8b55-871f-4c06-b8d2-542bc7e17a17)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/689deee0-35c1-427d-af6e-c489eaf1babf)

**Comment by salicius** [comment]
Con ISAM, la ricerca binaria non avviene sul file principale invece che sull'indice? 
Il terzo punto non si fa calcolando il log() del risultato del primo punto ?
